lib/Backends/Habana/Habana.cpp

```diff
@@ -1169,6 +1169,19 @@ HabanaBackend::compile(Function *F, const BackendOptions &opts) const {
       multiInputs.emplace_back(std::move(inputs));
       break;
     }
+    case Kinded::Kind::FusedRowwiseQuantizedSparseLengthsSumNodeKind: {
+      auto *RI = llvm::cast<FusedRowwiseQuantizedSparseLengthsSumNode>(&I);
+      std::vector<synTensor> inputs = {
+          tensors[RI->getData()].get(),
+          tensors[RI->getIndices()].get(),
+          tensors[RI->getLengths()].get(),
+      };
+      chk(synCreateGenericNode(inputs.data(), &tensors[RI].get(), inputs.size(),
+                               1, nullptr, "sparse_lengths_sum_u8_2D_f32_embed",
```

Comment:
have we already landed this kernel?

Reply:
This kernel already exists but not used. testing right now.